lib/rules/require-render-return.js

```diff
@@ -28,7 +28,8 @@ function create (context) {
       forbiddenNodes.forEach(el => {
         if (
           el.loc.start.line >= node.value.loc.start.line &&
-          el.loc.end.line <= node.value.loc.end.line
+          el.loc.end.line <= node.value.loc.end.line &&
+          node.value === el
```

Comment:
I think simple:
```
el.loc.start.line === node.value.loc.start.line
```
would be enough, or even just:
```
node.value === el
```
wdyt? (I was sceptic but looks like this kind of comparison works just fine for ASTs)

Plus I think that [this line](https://github.com/armano2/eslint-plugin-vue/blob/ffffa70df6f7bfa2967e048604e917f93c5f9ba2/lib/rules/require-render-return.js#L24)  should be moved to [extecuteOnFunctionsWithoutReturn](https://github.com/armano2/eslint-plugin-vue/blob/ffffa70df6f7bfa2967e048604e917f93c5f9ba2/lib/utils/index.js#L556):

```diff
- if (!isValidReturn()) {
+ if (!isValidReturn() && !node.expression) {
```